code/game/gamemodes/gang/gang.dm

```diff
@@ -1,7 +1,7 @@
 //gang.dm
 //Gang War Game Mode
 
-var/list/gang_name_pool = list("Clandestine", "Prima", "Zero-G", "Max", "Blasto", "Waffle", "North", "Omni", "Newton", "Cyber", "Donk", "Gene", "Gib", "Tunnel", "Diablo", "Psyke", "Osiron", "Sirius", "Sleeping Carp")
```

Comment:
@Robustin Yeah, but I was talking to @ChangelingRain 

I think he wanted you to remove the part where the sleeping carp could buy scrolls, too. In the gangtool code.